autora/michelle/css/style.css

```diff
@@ -0,0 +1,400 @@
+body{
+    margin:0;
+    font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+
+}
+
+.nav{
+background:#674584;
+position: fixed;
+width: 100%;
+top: 0;
+z-index: 99;
+
+
+}
+.main{
+    width:70%;
+    margin:0 auto;
+}
+.container{
+width: 80%;
```

Comment:
Você pode adicionar um max-width: tamanho desejado e um width:100% , assim quando a tela for desktop vai ter o tamanho desejado e quando diminuir a tela vai ter 100% do navegador independente do tamanho da viewport